packages/models-library/src/models_library/api_schemas_webserver/groups.py

```diff
@@ -54,14 +54,48 @@ class GroupAccessRights(BaseModel):
     )
 
 
-class GroupGet(OutputSchema):
-    gid: GroupID = Field(..., description="the group ID")
-    label: str = Field(..., description="the group name")
-    description: str = Field(..., description="the group description")
-    thumbnail: AnyUrl | None = Field(
-        default=None, description="url to the group thumbnail"
-    )
-    access_rights: GroupAccessRights = Field(..., alias="accessRights")
+class BasicGroupGet(OutputSchema):
```

Comment:
Based on other objects, should this not be?

```suggestion
class GroupGetBase(OutputSchema):
```